docs/website/contents/explanation/consensus-protocol.md

```diff
@@ -0,0 +1,267 @@
+
+This process involves two main components:
+
+- **Header envelope validation**: defined by the [`ValidateEnvelope`](#validateenvelope) class.
+- **Consensus protocol validation**: implemented indirectly via the `updateChainDepState` function of the [`ConsensusProtocol`](#consensusprotocol) class.
+The validity of a header is determined by examining its `ValidateView`.
+
+### Block Validity
+
+Full block validity involves both:
+
+- Applying the block to the ledger state.
+- Applying the header to the protocol state.
+
+This validation is a critical part of the [chain selection](#chain-selection) process.
+
+Although headers are already validated by the `ChainSync` client, they are re-validated during block application.
+This re-validation is necessary to update the `ChainDepState` within the extended ledger state.
+This step is expected to succeed, given that the header was previously validated by the `ChainSync` client.
+
+## Chain Ordering in Ouroboros
+
+The core rule in Ouroboros protocols (including Praos) is to prefer longer chains over shorter ones. This assumes that the honest majority of stake will produce denser chains.
+
+- A chain that extends the current one is always preferred.
+- If two chains are equally preferable, the node sticks with its current chain.
+
+In Praos and Shelley-based eras, if chains have the same length, tiebreaking [rules](https://github.com/intersectmbo/ouroboros-consensus/blob/010b374c54f4d2f485ab114f702db6ec3b7a8f95/ouroboros-consensus-protocol/src/ouroboros-consensus-protocol/Ouroboros/Consensus/Protocol/Praos/Common.hs#L108) are applied in this order:
+
+- 1. **Operational Certificate Issue Number**:
+If both chain tips are from the same issuer and slot, the one with the higher `opcert` issue number is preferred.
+This allows a stake pool operator to replace a compromised hot key with a new one and still have their blocks take precedence.
+
+- 2. **VRF Tiebreaker**:
+If the `opcert` check is inconclusive (which is common when two pools are elected in the same slot), the chain with the lower VRF value at its tip is preferred.
+This avoids always picking the first block to arrive, which could encourage centralization to reduce latency (see ["The Frankfurt Problem"](https://github.com/IntersectMBO/ouroboros-consensus/blob/40d77fdd74a9b2b2a1d112a0b836b5cb8026c88c/ouroboros-consensus-protocol/src/ouroboros-consensus-protocol/Ouroboros/Consensus/Protocol/Praos/Common.hs#L227)). The VRF value used for tiebreaking (non-range extended) is uncorrelated to the leader VRF value and typically results in a uniformly random decision. Depending on the `ChainOrderConfig` for Praos, there are two [flavors](https://github.com/intersectmbo/ouroboros-consensus/blob/010b374c54f4d2f485ab114f702db6ec3b7a8f95/ouroboros-consensus-protocol/src/ouroboros-consensus-protocol/Ouroboros/Consensus/Protocol/Praos/Common.hs#L75) that determine when this tie-breaker comparison takes place.
```

Comment:
centralization -> geographic centralization?

centralization -> network centralization?

centralization -> physical collocation?

Reply:
From the three options above, I'd go with either "geographic centralization" or "physical collocation". I think it's better to mention centralization since the contrast with the aspiration of desentralization is more evident (or immediate).